README.md

```diff
@@ -1,181 +1,196 @@
+# LTI 1.3 Advantage Library
 
-| **Note** : If you are looking for the example tool that uses this library, it has been moved into its own repo https://github.com/IMSGlobal/lti-1-3-php-example-tool |
-| --- |
+A library used for building IMS-certified LTI 1.3 tool providers in PHP.
 
-# LTI 1.3 Advantage Library
-This code consists of a library for creating LTI tool providers in PHP.
+## Installation
 
-# Library Documentation
+Run:
 
-## Importing the library
-### Using Composer
-Add the following to your `composer.json` file
-```json
-"repositories": [
-    {
-        "type": "vcs",
-        "url": "https://github.com/IMSGlobal/lti-1-3-php-library"
-    }
-],
-"require": {
-    "imsglobal/lti-1p3-tool": "dev-master"
-}
+```bash
+composer require packbackbooks/lti-1p3-tool
 ```
-Run `composer install` or `composer update`
-In your code, you will now be able to use classes in the `\IMSGlobal\LTI` namespace to access the library.
 
-### Manually
-To import the library, copy the `lti` folder inside `src` into your project and use the following code at the beginning of execution:
+In your code, you will now be able to use classes in the `Packback\Lti1p3` namespace to access the library.
+
+### Configuring JWT
+
+Add the following when bootstrapping your app.
+
 ```php
-require_once('lti/lti.php');
-use \IMSGlobal\LTI;
+use Firebase\JWT\JWT;
+
+JWT::$leeway = 5;
 ```
 
-## Accessing Registration Data
+### Data Storage
+
+This library uses three methods for storing and accessing data: cache, cookie, and database. All three must be implemented in order for the library to work. You may create your own custom implementations so long as they adhere to the following interfaces:
+
+- Packback\Lti1p3\Interfaces\Cache
+- Packback\Lti1p3\Interfaces\Cookie
+- Packback\Lti1p3\Interfaces\Database
+
+Cache and Cookie storage have legacy implementations at Packback\Lti1p3\ImsStorage if you do not wish to implement your own. However you must implement your own database.
```

Comment:
Just general thoughts here - are the legacy `Packback\Lti1p3\ImsStorage` used by default? I don't know what these are doing, but generally it's good for a package to start with "batteries included" and then explain more advanced usages like implementing your own storage mechanism.

Reply:
They're not used by default. Honestly, I'm hesitant to have them used out of the box. They're not the most flexible and make a lot of assumptions about implementation. For example, Laravel's Cache and Cookie stores are completely incompatible with the IMS implementations. If you drop them into a fresh Laravel install and try to use them, it'll 500.